creator-node/src/utils.js

```diff
@@ -216,8 +220,28 @@ async function rehydrateIpfsDirFromFsIfNecessary (req, dirHash) {
   }
 }
 
+async function fetchCIDFromOtherCreatorNodes (req, multihash, expectedStoragePath) {
+  const libs = req.app.get('audiusLibs')
+  const nodes = await libs.ethContracts.ServiceProviderFactoryClient.getServiceProviderList('creator-node')
```

Comment:
should we check from chain here or discprov? discprov means adding a new route

Reply:
add to req object